### ISSUE TITLE
Failing to deploy an old Ruby on Rails application

### ISSUE THREAD

**Issue by philiplb** [flynn/flynn#3789]
Hi,
I have an old Ruby on Rails application which is failing to deploy like this:

>        ...
>        Installing tzinfo 0.3.29
>        Your Gemfile.lock is corrupt. The following gems are missing from the
>        DEPENDENCIES section: 'archive-tar-minitar' 'hoe' 'nokogiri' 'rcov'
>        !
>        !     Failed to install gems via Bundler.
>        !

I have another Rails app deployed from that era with success, so in general, it works. The Gemfile of this failing app looks like this:

```
source 'http://rubygems.org'

# default gems
gem 'rails', '3.0.20'
#gem 'sqlite3-ruby', :require => 'sqlite3'

ruby "1.9.3"

gem 'rvm-capistrano'

# deploy with Capistrano
gem 'capistrano'
gem 'capistrano-ext'

# do not use mysql driver but mysql2 driver for full unicode support
gem "mysql2", "0.2.11"

# paperclip
gem 'aws-s3', "0.6.3"
gem "paperclip", "2.3.8"

# adds deleted_at support
gem 'paranoia'

gem 'inherited_resources'
gem 'inherited_resources_views'

gem 'haml-rails'
gem 'jquery-rails'

gem 'newrelic_rpm', '3.1.2'
```

The issued bundle install command works just fine on a regular linux VM.
Do you have any clues here?

**Comment by titanous** [comment]
Can you try updating the `bundler` gem to the latest version, run `bundle install`, commit `Gemfile.lock` and push again? If that doesn't work you may need to delete `Gemfile.lock` and redo the last two steps.

**Comment by philiplb** [comment]
Hi,
thank you for the steps. That worked (Beside some random crashes now on booting the app, but it gets deployed, so this issue is resolved.)
What I did, for the record:
bundler was in the version 1.7.6 before. I ran "gem update bundler" which got me to the version 1.13.7. I deleted the Gemfile.lock and ran "bundle install --no-deployment". This got me an updated Gemfile.lock which I committed. Afterwards, the "git push flynn master" worked fine. :)
